README.md

```diff
@@ -0,0 +1,45 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# SFD2022 Sample Design
+
+Sample Design of Imaginaty Software Freedom Day 2022 event
+
+## Built With
+
+- HTML5
+- CSS3
+- JavaScript
+
+## Getting Started
+
+This is a simple static website made with HTML5, CSS3 and JavaScript. There is not specific requirements for building the project.
+
+## Development
+
+In order to check linters locally, run `npm install` to get required packages and then run `npm run linter` to run all linters (eslint, stylelint, hint). For Lighthouse, either user chromium based browsers or run a live server and use its user on this command: `npx lighthouse <URL>`.
```

Comment:
- [ ] The second requirement is that you should deploy your website using Github pages. Let's deploy the website using Github pages and put the live version link in the readme file 😀👍

Reply:
Thanks. I have already deployed but forgot to add the link to README. Thanks for pointing out.

Reply:
@ahangarha yes It also happens many times with me 😉